sdk/search/search-documents/README.md

```diff
@@ -529,6 +529,28 @@ async function main() {
 main();
 ```
 
+#### Get a list of existing indexers in the service
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
+
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
+
+async function main() {
+  let listOfIndexers = await client.listIndexers({
+    select: ['name', 'skillsetName', 'dataSourceName']
```

Comment:
do we think selecting a subset of indexer properties is common? The object doesn't seem big enough that this would be a common scenario

Reply:
Done (in the next commit).